README.md

```diff
@@ -1,75 +1,36 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# HELLO MICROVERSE PROJECT
 
-> Description the project.
+> A Microverse project for beginer micronauts.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+- HTLM
+- CSS
+- Linter
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+## Author
```

Comment:
- [ ] Here is where you explain how the person can clone and the project on there machine. please see the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) for further understandings